src/main/java/Legin.java

```diff
@@ -0,0 +1,232 @@
+import exception.LeginMissingParamsException;
+import exception.LeginEmptyTaskException;
+import exception.LeginIndexOutOfBoundsException;
+import exception.LeginInvalidCommandException;
+import task.Deadline;
+import task.Event;
+import task.Task;
+import task.Todo;
+import java.io.IOException;
+import java.util.ArrayList;
```

Comment:
Good job on providing explicit imports. Perhaps you would want to order them in a collated order e.g. java.util.ArrayList is together with java.util.Scanner, since both comes from java.util package.
Note that this is part of Java coding standard - Intermediate.